### PR TITLE
refactor: initial styling of zApp (extract temporary inline styling - add scss)

### PR DESCRIPTION
**Pull request by domw30**
[Associated Notion Card](https://www.notion.so/zerotech/dApp-Team-Dash-78b824c0cd0e46c59c19dfe103ae929a?p=6469169807a64650b892211162840eaf&pm=s)

-----

## 1. Pull request checklist

- [x] Notion card has been moved to the Code Review column
- [x] Notion card has a link to this PR
- [x] A reviewer has been assigned to the Notion card

## 2. PR type

    - Some Refactoring
    - Feature - Initial styling for zApp-NFTs

## 3. What is the old behaviour?
Inline styles were added as a temporary measure for data layer development. This project aims to remove all uses of inline styling and extract to scss files - using zUI to import variables/available components. 

## 4. What is the new behaviour?
scss files created and initial round of styling added

### CODE REVIEW

**Review by colbr — APPROVED** [review]

File: src/features/view-subdomains/SubdomainTable/SubdomainTable.tsx [hunk 1]
```diff
@@ -39,16 +48,28 @@ export const SubdomainTable: FC<SubdomainTableProps> = ({
...
-					justifyContent: 'flex-end',
-					color: 'black',
-				}}
-			>
-				<button onClick={() => changeView(false)}>Row</button>
-				<button onClick={() => changeView(true)}>Grid</button>
+			{/* todo: leave or add controls to table component */}
+			<div className={styles.Controls}>
+				<div className={styles.SearchBarContainer}>
+					<SearchBar
```
Comment (domw30) [inline src/features/view-subdomains/SubdomainTable/SubdomainTable.tsx:54]: Search functionality to be wired up in a different PR - shell component for now

File: src/features/domain-preview/DomainPreview.tsx [hunk 2]
```diff
@@ -26,36 +32,56 @@ export const DomainPreview: FC<DomainPreviewProps> = ({
...
-
-			{description && <p>{description}</p>}
-
-			{href && (
-				<Link style={{ background: 'none', color: '#52cbff' }} to={href}>
-					{'View Domain NFT ->'}
-				</Link>
-			)}
+		<div className={styles.Container}>
+			{/* todo: media asset component */}
```
Comment (domw30) [inline src/features/domain-preview/DomainPreview.tsx:42]: comments added to reflect these are just placeholder components for now to help with styling

File: src/features/view-subdomains/SubdomainTable/SubdomainTable.tsx (hunk 1 above)
Comment (colbr) [reply src/features/view-subdomains/SubdomainTable/SubdomainTable.tsx:54]: Nice! Don't worry about search for now - it will be built into the table component.

**Review by colbr — APPROVED** [review]
Nice! Feels good to cut out those inline-styles - looking forward to the next iterations.
